package.json

```diff
@@ -25,14 +25,14 @@
   "dependencies": {
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^2.0.0",
+    "ember-cli-babel": "^6.0.0",
```

Comment:
It looks like the docs indicate this should be ^6.8.0, or does that matter?

Reply:
I just moved the dependency for this PR to fix a deprecation warning. The next step in this sequence is to actually use the new module API, and that PR will update all the dependencies, including ember-cli-babel.

Reply:
On further thought, I'll probably split that into two steps (updating dependencies, then using the new module API), but either way the next PR will take care of updating this dependency.